server/src/routes/v2/organisations.js

```diff
@@ -69,6 +69,24 @@ module.exports = [
             return orgs
         },
     },
+    {
+        method: 'GET',
+        path: '/v2/organisations-with-apps',
```

Comment:
not sure about the route name - the API is not RESTful anyhow, so went for simplicity

Reply:
I would expect this to be under `/organisations`. And maybe a resource under that?